pages/etherpad/actions/CreateNewPad.js

```diff
@@ -0,0 +1,89 @@
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { styled } from 'styled-components';
+
+import LoadingSpinnerInline from '../../../components/UI/LoadingSpinnerInline';
+import logger from '../../../lib/Logging';
+import ErrorMessage from '@/components/UI/ErrorMessage';
+import { Input } from '@/components/UI/shadcn/Input';
+import { Button } from '@/components/UI/shadcn/Button';
+import { Switch } from '@/components/UI/shadcn/Switch';
+
+const SwitchContainer = styled.div`
+    display: flex;
+    gap: var(--margin);
+    align-items: center;
+    justify-content: space-between;
+`;
+
+export default function CreateNewPad({ createPadAndOpen, isMyPadsApiEnabled, callbackDone }) {
+    const { t } = useTranslation('etherpad');
+
+    const [padName, setPadName] = useState('');
+    const [passwordProtected, setPasswordProtected] = useState(false);
+    const [password, setPassword] = useState('');
+    const [validatePassword, setValidatePassword] = useState('');
+    const [isLoading, setIsLoading] = useState(false);
+    const [errorMessage, setErrorMessage] = useState('');
+
+    const createPad = async () => {
+        if (password !== validatePassword) {
+            setErrorMessage('Passwords do not match');
+
+            return;
+        }
+
+        setIsLoading(true);
+        setErrorMessage('');
+        const createPad = await createPadAndOpen(padName, passwordProtected ? 'private' : 'public', password).catch((error) => {
+            logger.debug(error);
+            setErrorMessage('Something went wrong, please try again.');
+        });
+        setIsLoading(false);
+        if (!createPad) return;
+        callbackDone && callbackDone();
+        setPadName('');
+    };
+
+    return (
+        <form
+            className="mb-8 [&>*+*]:mt-4"
```

Comment:
Instead of using this selector, how about adding a `mt-4` to every component below? I think that would be the more correct "Tailwind"-approach.